myfitnesspal/day.py

```diff
@@ -16,7 +16,7 @@ def __getitem__(self, value):
         raise KeyError("No meal named '%s' exists for this date" % value)
 
     def keys(self):
-        keys = ''
+        keys = []
```

Comment:
Weird -- I don't know how I never noticed this before; this couldn't have been working earlier, right?